modules/ROOT/pages/configure-monitoring-cloudhub.adoc

```diff
@@ -97,7 +112,7 @@ Use the Anypoint CLI and set the following property during your CI/CD deployment
 
 == Performance Impact
 
-Enabling Monitoring can impact CPU utilization and memory utilization, which varies depending upon specific application configuration. MuleSoft recommends that you test any performance impact in your pre-production environment prior to enabling monitoring in your production environment to ensure optimal performance and sizing.
+Enabling monitoring can impact CPU utilization and memory utilization, which varies depending upon specific application configuration. You should test any performance impact in your pre-production environment prior to enabling monitoring in your production environment to ensure optimal performance and sizing.
```

Comment:
Hmmm. I kind of worry about having this very important requirement isolated under its own title, rather than part of the task, and I wonder if there is some way we make this part of the intro text or a final step to the task or text after the final step of the task. 
Should we even consider retitling to Enable and Test...?

Reply:
We put it pretty low since it's a rather small percentage of customers with a pretty small impact overall, IIRC.